Manifests/ManagedPreferencesApplications/com.brave.Browser.plist

```diff
@@ -15,7 +15,7 @@
 	<key>pfm_interaction</key>
 	<string>combined</string>
 	<key>pfm_last_modified</key>
-	<date>2019-12-10T17:48:00Z</date>
+	<date>2020-04-08T20:22:48Z</date>
```

Comment:
If possible, can you also add the additional v80 & v81 Chrome prefs I added yesterday?  Was planning to tackle adding them to Brave today, but since you've already started ... :)

Reply:
You'll find this implemented on commit 025a55d.